router/pump.go

```diff
@@ -118,15 +130,17 @@ func (p *LogsPump) Run() error {
 }
 
 func (p *LogsPump) pumpLogs(event *docker.APIEvents, backlog bool) {
+	debug("pump.pumpLogs() event.ID:", event.ID)
+	debug("pump.pumpLogs() event.Actor.ID:", event.Actor.ID)
 	id := normalID(event.ID)
```

Comment:
using `event.Actor.ID` here caused a panic at startup. 
Additional debugging shows `event.Actor.ID` is null for some reason in the startup case
`ef13d311a6c3` is the logspout container
```
2016/04/20 18:59:51 pump.pumpLogs() event.ID: ef13d311a6c3
2016/04/20 18:59:51 pump.pumpLogs() event.Actor.ID:
2016/04/20 18:59:51 pump: json: cannot unmarshal array into Go value of type docker.Container
```